src/plugins/ini/ini.c

```diff
@@ -84,6 +84,8 @@ static int iniKeyToElektraKey (void *vhandle, const char *section, const char *n
 
 	keyAddBaseName (appendKey, name);
 
+	if(value == NULL)
+		keySetMeta(appendKey, "ini/empty", "");
```

Comment:
Can a null key be used unambiguously? If yes, it should be preferred. There are already 3 states of emptyness (key not present, value not present, string emtpy), we should not over-complicate everything.